bcipy/signal/model/pca_rda_kde/pca_rda_kde.py

```diff
@@ -42,21 +49,31 @@ def fit(self, train_data: np.array, train_labels: np.array) -> SignalModel:
         rda_index = 1  # the index in the pipeline
         model.pipeline[rda_index].lam = arg_cv[0]
         model.pipeline[rda_index].gam = arg_cv[1]
-        _, sc_cv, y_cv = cost_cross_validation_auc(
+        tmp, sc_cv, y_cv = cost_cross_validation_auc(
```

Comment:
Why this function returns negative AUC? (`-tmp`)

Reply:
The hyperparam search in `nonlinear_opt` function is using a generic optimization method that minimizes a function `scipy.optimize.fmin_cobyla(...)`. In order to maximize `AUC`, it just minimizes `-AUC`.

That optimization function `nonlinear_opt` makes use of `cost_cross_validation_auc`, which is why `cost_cross_validation_auc` needs to return negative AUC.

Reply:
Thank you so much for the clarification!